README.md

```diff
@@ -1,17 +1,32 @@
-# mcrouter
-[![Build Status](https://travis-ci.org/evertrue/mcrouter-cookbook.svg)](https://travis-ci.org/evertrue/mcrouter-cookbook)
```

Comment:
It wasn't working anymore, the original maintainer is aware of this and he's trying to fix it on his own branch.